vllm/attention/layer.py

```diff
@@ -485,6 +485,7 @@ def unified_attention_fake(
     mutates_args=[],
     fake_impl=unified_attention_fake,
     dispatch_key=current_platform.dispatch_key,
+    tags=(torch._C.Tag.cudagraph_unsafe, ),
```

Comment:
Will this be an issue for the current integration?

Reply:
no. it's a noop when not using inductor graph partition.